packages/frontend-api/build.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="frontend_api">
+    <property name="path.openssl.executable" value="openssl"/>
+    <property name="path.defuse-key-generate.executable" value="${path.bin}/generate-defuse-key"/>
+
+    <target name="frontend-api-generate-new-keys" description="Enable frontend API for all domains.">
```

Comment:
Shouldn't be this target under production protection?

Reply:
28b24330b4eaeebcc2d1ffce03d4d0f5a448cbab